.github/workflows/release.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Install Helm
         uses: azure/setup-helm@v3
         with:
-          version: v3.10 # renovate: datasource=github-releases depName=helm packageName=helm/helm
+          version: v3.10.3 # renovate: datasource=github-releases depName=helm packageName=helm/helm
```

Comment:
Can we omit the version so we're always using the stable version?

https://github.com/Azure/setup-helm#example

Reply:
I guess there is a pro and con to it as there is a note there about the API rate limit, which if the runner hits, it defaults back to their hard coded version, which might not be the latest. If we control the version, it's guaranteed that it won't run it with an older version then the previous build because of it.

Reply:
@nitrocode Should I update it then?

Reply:
I defer to the other @runatlantis/maintainers here

Reply:
I added the other option as a suggestion.